src/convertToObject.js

```diff
@@ -6,7 +6,32 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const mainObj = {};
+
+  const workArr = sourceString.split(';');
+
+  const cleanArr = workArr
+    .map(function (item) {
+      const result = item.replace(/\n/g, '').trim();
+
+      return result;
```

Comment:
You don't need to create a new variable `result` here. You can return the result of the `replace` and `trim` operations directly.